### PR TITLE
Apps readme files

### PR DESCRIPTION
**Pull request by flashdesignory**
Readmes for all apps.

Will add more details as we integrate with Speedometer.

@kara

### CODE REVIEW

File: apps/complex-dom/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+A collection of todomvc apps wrapped in a complex dom.
+
+-   [todomvc-angular-complex](./todomvc-angular-complex/README.md)
+-   [todomvc-backbone-complex](./todomvc-backbone-complex/README.md)
+-   [todomvc-es5-complex](./todomvc-es5-complex/README.md)
+-   [todomvc-es6-webpack-complex](./todomvc-es6-webpack-complex/README.md)
+-   [todomvc-jquery-complex](./todomvc-jquery-complex/README.md)
+-   [todomvc-lit-complex](./todomvc-lit-complex/README.md)
+-   [todomvc-preact-complex](./todomvc-preact-complex/README.md)
+-   [todomvc-reac-complex](./todomvc-react-complex/README.md)
```
Comment (kara) [inline apps/complex-dom/README.md:12]:
```suggestion
-   [todomvc-react-complex](./todomvc-react-complex/README.md)
```

File: apps/charts/charts-chartjs/README.md [hunk 2]
```diff
@@ -23,42 +23,15 @@ npm run dev
...
-### [Observable Plot](https://github.com/observablehq/plot)
-
-You can load this benchmark with the `/observable-plot.html` page, for example
-http://localhost:5173/observable-plot.html if you run it locally or
-http://localhost:7000/resources/charts/dist/observable-plot.html in the
-context of the speedometer.
-
-Observable Plot is D3-based and outputs SVG.
-
-When run in development mode, the page will automatically execute the 3
```
Comment (kara) [inline apps/charts/charts-chartjs/README.md:49]: Curious why this was removed - no longer accurate?

**Review by kara — APPROVED** [review]
LGTM, aside from the typo

File: apps/charts/charts-chartjs/README.md (hunk 2 above)
Comment (flashdesignory) [reply apps/charts/charts-chartjs/README.md:49]: charts and editors were basically two apps in one folder. 
I separated them out and only kept in each readme what is relevant to its particular app.

Comment (kara) [reply apps/charts/charts-chartjs/README.md:49]: Makes sense!

**Review by kara — APPROVED** [review]

**Pull request merged**